openfga_sdk/telemetry/attributes.py

```diff
@@ -69,21 +85,44 @@ def prepare(
 
         if attributes is not None:
             for attribute, value in attributes.items():
-                if isinstance(attribute, TelemetryAttribute):
-                    if filter is not None and attribute not in filter:
-                        continue
+                if value is None:
+                    continue
+
+                if isinstance(attribute, str):
+                    attribute = attribute.lower().replace("-", "_").replace(".", "_")
```

Comment:
What is this for?

Reply:
```python
def prepare(
    self,
    attributes: dict[TelemetryAttribute | str, str | int] | None,
    filter: list[TelemetryAttribute | str] | None = None,
) -> dict[str, str | int]:
```

`attributes` can either be a TelemetryAttribute instance or a string. This allows us to provide `fga-client.request.client_id` as the attribute key and properly resolve it to an instance of the `fga_client_request_client_id` TelemetryAttribute.